makejdk.sh

```diff
@@ -338,22 +344,32 @@ buildAndTestOpenJDKViaDocker()
   # Keep is undefined so we'll kill the docker image
 
   if [[ "$KEEP" == "true" ]] ; then
-     if [ "$(docker ps -a | grep -c openjdk_container)" == 0 ]; then
+     if [ "$(docker ps -a | grep -c $CONTAINER)" == 0 ]; then
          echo "${info}No docker container found so creating '$CONTAINER' ${normal}"
          docker build -t "$CONTAINER" "$PATH_BUILD"
+         if [[ "$ALTERNATE_VARIANT" != "" && -f $PATH_BUILD/Dockerfile-$ALTERNATE_VARIANT ]]; then
+             CONTAINER=${CONTAINER}-${ALTERNATE_VARIANT}
+             echo Building variant $ALTERNATE_VARIANT
+             docker build -t "$CONTAINER" -f "$PATH_BUILD/Dockerfile-$ALTERNATE_VARIANT" "$PATH_BUILD"
+         fi
      fi
   else
      echo "${info}Building as you've not specified -k or --keep"
      echo "$good"
      docker ps -a | awk '{ print $1,$2 }' | grep $CONTAINER | awk '{print $1 }' | xargs -I {} docker rm -f {}
      docker build -t "${CONTAINER}" "${PATH_BUILD}" --build-arg OPENJDK_VERSION="${OPENJDK_VERSION}"
+     if [[ "$ALTERNATE_VARIANT" != "" && -f $PATH_BUILD/Dockerfile-$ALTERNATE_VARIANT ]]; then
```

Comment:
Not sure if this small duplicate chunk can be extracted?

Reply:
Good feedback, thanks. i can extract into a "buildDockerContainer" function that avoids most of the duplication. will add a commit to show that (after I've tested it :) ) for review